packages/itwin/grouping-mapping-widget/src/test/QueryBuilder.test.ts

```diff
@@ -0,0 +1,186 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { Query, QueryBuilder, QueryClass } from "../widget/components/QueryBuilder";
+import { assert } from "chai"
+
+describe("QueryBuilder", () => {
+  describe("buildQueryString()", () => {
+
+    beforeEach(() => {
+    });
+
+    afterEach(() => {
+    });
+
+    const testBuildQueryString = (query: Query|undefined, expectedResult: string) => {
```

Comment:
When writing types with `|` - add spaces around the pipe symbol.

Reply:
Fixed